bitnami/minio/templates/servicemonitor.yaml

```
@@ -1,5 +1,5 @@
 {{- if .Values.metrics.serviceMonitor.enabled }}
-apiVersion: monitoring.coreos.com/v1
+apiVersion: {{ .Values.metrics.serviceMonitor.apiVersion | default "monitoring.coreos.com/v1" }}
```

Minor improvement, for consistency with the rest of the codebase:
```suggestion
apiVersion: {{ default "monitoring.coreos.com/v1" .Values.metrics.serviceMonitor.apiVersion }}
```